kernel/src/device/tdxguest/mod.rs

```diff
@@ -82,8 +83,7 @@ impl FileIo for TdxGuest {
 fn handle_get_report(arg: usize) -> Result<i32> {
     const SHARED_BIT: u8 = 51;
     const SHARED_MASK: u64 = 1u64 << SHARED_BIT;
-    let user_space = get_current_userspace!();
-    let user_request: TdxReportRequest = user_space.read_val(arg)?;
+    let user_request: TdxReportRequest = get_current_userspace!().read_val(arg)?;
```

Comment:
Assign the return value of `get_current_userspace` to a local variable to avoid calling it repeatedly.

Reply:
The temporary `Task` object returned by `ostd::task::Task::current().unwrap()` is released when the macro call ends. Therefore, we cannot bind variables to it. So I used `ostd::task::Task::current().unwrap()` directly.